templates/cisco_ios_show_ip_ospf_database_router.textfsm

```diff
@@ -0,0 +1,48 @@
+Value Filldown ROUTER_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown PROCESS_ID (\d+)
+Value Filldown AREA (\d+\.\d+\.\d+\.\d+|\d+)
+Value Filldown LSA_AGE (\d+)
+Value Filldown LSA_OPTIONS (.*)
+Value Filldown LSA_TYPE (.*)
+Value Filldown LSA_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown LSA_ADV_ROUTER (\d+\.\d+\.\d+\.\d+)
+Value Filldown LSA_SEQ_NUMBER (\S+)
+Value Filldown LSA_CHECKSUM (\S+)
+Value Filldown LSA_LENGTH (\d+)
+Value Filldown LSA_NUM_LINKS (\d+)
+Value LS_LINK_TYPE ((a Stub Network)|(a Transit Network)|(another Router \(point-to-point\)))
+Value LS_LINK_ID (\d+\.\d+\.\d+\.\d+)
+Value LS_LINK_DATA (\d+\.\d+\.\d+\.\d+)
+Value LS_MTID_METRICS (\d+)
+Value LS_TOS_0_METRICS (\d+)
+
+Start
+  ^\s+OSPF Router with ID \(${ROUTER_ID}\) \(Process ID ${PROCESS_ID}\)
```

Comment:
Do you mind converting all the static spaces to `\s+`. This just helps to not break the template if the vendor changes the spacing.

Reply:
Fixed via 87a1f37